vizro-core/src/vizro/_vizro.py

```diff
@@ -88,7 +88,7 @@ def _append_styles(walk_dir: str, url_prefix: str) -> Tuple[List[str], List[str]
             path = os.path.join("/" + url_prefix, base, f) if base else os.path.join("/" + url_prefix, f)
             extension = os.path.splitext(f)[1]
             if extension == ".js":
-                _vizro_js.append(path)
+                _vizro_js.append({"src": path, "type": "module"})
```

Comment:
@huong-li-nguyen Can you double check if this is in line with what we previously intended. Was not so involved in this

Reply:
What does this do? As far as I remember we initially implemented this as the assets couldn't be read in properly on some of the hosted versions. For me it's fine as this works as before and the js files live in the static folder. 

@Anna-Xiong - can you double-check Petars changes to the initial function you wrote?

Reply:
@Anna-Xiong Or could you also have a look?

Reply:
This change was made to enable import/export features in `js` scripts. Without setting those scripts as modules (with `"type": "module"`) I had to push all the `js` files into the one big file (which doesn't look pretty).

Reply:
I am not so sure about js here, have you tested adding js @petar-qb ? 
The thing is that in source code i did not see adding this type and module, but maybe the js has been created into another object

Reply:
@maxschulz-COL @petar-qb - did you guys follow up on this one before the merge? Otherwise please double-check

Reply:
This function returns only the `css` and `js` files from the `src/vizro/static/` folder. There won't be any problems because `src/vizro/static` contains set of css files and only javascript files added by me. A dashboard creator cannot customise `src/vizro/static` folder because additionally added `css/js` files (added by dashboard creator) are added through the `assets` folder instead (`src/vizro/assets/`).